examples/controls_example/public/react_controls/control_panel.scss

```diff
@@ -10,14 +10,16 @@
     border-radius: $euiBorderRadius !important;
   }
 
-  &--label {
-    @include euiTextTruncate;
+  &--labelWrapper {
     max-width: 40%;
-    background-color: transparent;
-    border-radius: $euiBorderRadius;
 
-    margin-left: 0 !important;
-    padding-left: 0 !important;
+    .controlPanel--label {
+      @include euiTextTruncate;
+      background-color: transparent;
+      border-radius: $euiBorderRadius;
+      margin-left: 0 !important;
+      padding-left: 0 !important;
+    }
```

Comment:
This fixes a style error where the truncation was happening **before** the `max-width` applied, so labels were getting cut off too early on the "small" control size:

| Before | After |
|--------|--------|
| Cell | ![image](https://github.com/elastic/kibana/assets/8698078/3ae9ae35-0a53-401b-b92d-fcfe4de18d83) | 

I verified that the truncation now happens at the **same point** that it does for the old controls.